github/ipfs.yml

```diff
@@ -5809,12 +5772,10 @@ repositories:
     squash_merge_commit_title: COMMIT_OR_PR_TITLE
     teams:
       admin:
-        - admin
         - ipdx
```

Comment:
@ipfs/ipdx do you need admin here (recognizing the @galargh has org admin permissions anyway, but trying to understand what permissions are necessary for you to get your work done.

Reply:
Nope, I think we can get rid of this one here.

```suggestion
```